youtube_dlc/extractor/zee5.py

```diff
@@ -0,0 +1,85 @@
+# coding: utf-8
+from __future__ import unicode_literals
+
+from .common import InfoExtractor
+
+import requests
+
+token_url = "https://useraction.zee5.com/tokennd"
+search_api_endpoint = "https://gwapi.zee5.com/content/details/"
+platform_token = "https://useraction.zee5.com/token/platform_tokens.php?platform_name=web_app"
+stream_baseurl = "https://zee5vodnd.akamaized.net"
+
+headers = {
+    "User-Agent": "Mozilla/5.0 (Linux; Android 10; SM-J400F) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/81.0.4044.138 Mobile Safari/537.36",
+    "Referer": "https://www.zee5.com",
+    "Accept": "*/*",
+    "Accept-Encoding": "gzip, deflate, br",
+    "Accept-Language": "en-US,en;q=0.9",
+    "Origin": "https://www.zee5.com",
+    "sec-fetch-dest": "empty",
+    "sec-fetch-mode": "cors",
+    "sec-fetch-site": "same-site",
+}
+
+
+class Zee5IE(InfoExtractor):
+    _VALID_URL = r'https?://(?:www\.)?zee5\.com/.*/(?P<id>.*)'
+    _TEST = {
+        'url': 'https://www.zee5.com/movies/details/krishna-the-birth/0-0-63098',
+        # 'md5': 'cd5abd10fab020dbb633cb8e94df0e12',
```

Comment:
Dont disable this

Reply:
uncommenting this results in tests not passing.

Reply:
the md5 given should be correct. I'll fix it